src/storage/src/store.rs

```diff
@@ -271,6 +282,21 @@ pub trait LocalStateStore: StateStoreRead + StateStoreWrite + StaticSendSync {
     fn advance_write_epoch(&mut self, _new_epoch: u64) -> StorageResult<()> {
         unimplemented!()
     }
+
```

Comment:
Is it better to move this method to `StateStoreRead`, since this is a read related method.

Reply:
That was what I did previously but later I realized that this method should only be used by **streaming** executor to check whether a key range exists on the **latest** data. That is why I put it in `LocalStateStore`.